### ISSUE TITLE
ts-node commond not found  after update docker image

### ISSUE THREAD

**Issue by ditunes** [Chatie/wechaty#492]
alfter pull new docker image i can't start my bot ,the error as follows
```
docker run -ti --volume="$(pwd)":/bot --rm zixia/wechaty src/index.ts
```

but if i used old docker image it's ok ,what happened?

**Comment by ditunes** [comment]
if i pull the  image  onbuild  everthing is ok

**Comment by zixia** [comment]
Did you pull the latest image from Docker Hub?

I noticed that there's something wrong with the latest code with Docker, but there should not be deployed yet because it will always fail the CI testing so no Deployment should be done.

**Comment by zixia** [comment]
Sorry, this problem is introduced by https://github.com/Chatie/wechaty/commit/79a2517c0842b16da3595b2c7f79bcdc554a48bb#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

I forget why it happens, maybe it's a misoperation made by me.

**Comment by zixia** [comment]
Should be fixed after v0.8.12

**Comment by zixia** [comment]
Should be fixed after v0.8.12
